bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoCoffeeHandler.java

```diff
@@ -210,9 +207,9 @@ private synchronized void onSubscription() {
             logger.debug("Checking WeMo GENA subscription for '{}'", this);
 
             String subscription = "deviceevent1";
-            if ((subscriptionState.get(subscription) == null) || !subscriptionState.get(subscription).booleanValue()) {
+            if (subscriptionState.get(subscription) == null) {
```

Comment:
Isn't this changing the behavior? Why are you changing this?

Reply:
No, I don't think so. subscriptionState is defined as boolean and only set true or false in the binding. So in my opinion, this check was superflient. Don't know whay it was added.

Reply:
Are the `subscriptionState` values ever used anywhere? If not then you should make `subscriptionState` a Set instead of a Map.

Reply:
The values are used in onServiceSubscribed(), onSubscription() and removeSubscription()